R/mobr_boxplots.R

```diff
@@ -289,9 +395,12 @@ plot.mob_stats = function(mob_stats,
                   main = panel_title, ...)
           if (multipanel)
```

Comment:
Yeah, that is true. As far as I remember Xiao inplemented the plotting for the different panels. Jon requested to add even more indices (rarefied richness for different N, coverage standardized richness for different coverage levels). So I prefer revising the plotting functions, once we selected the indices.